manticore/models.py

```diff
@@ -0,0 +1,80 @@
+from .utils.helpers import issymbolic
+from .core.smtlib.solver import solver
+from .core.smtlib.operators import ITEBV, ZEXTEND
+
+
+def _find_zero(cpu, constrs, ptr):
```

Comment:
I don't think this belongs in `Cpu` (i think `read_string` should though, like `read_int` & co). I think this is a valid place for it; might even remove the leading underscore as it can be an externally-useful function.